### PR TITLE
ceph.spec.in: use %autosetup macro

### PR DESCRIPTION
**Pull request by ktdreyer**
New in rpm 4.11,
http://www.rpm.org/wiki/PackagerDocs/Autosetup

This makes it a bit easier to incorporate patches downstream.

### CODE REVIEW

**Comment by ktdreyer** [comment]
CC'ing @smithfarm in case he wants to review

File: ceph.spec.in [hunk 1]
```diff
@@ -619,7 +619,7 @@ python-cephfs instead.
 # common
 #################################################################################
 %prep
-%setup -q
+%autosetup -p1
```
Comment (smithfarm) [inline ceph.spec.in]: Why -p1 when there are no patches?

**Comment by smithfarm** [comment]
@ktdreyer I checked and all the SUSE versions we are supporting have at least version 4.11. I guess you want to the `-p1` so you can apply downstream patches automatically, and it doesn't hurt the upstream build to have it there, so

`Reviewed-by: Nathan Cutler <ncutler@suse.com>`

**Comment by ktdreyer** [comment]
Exactly, it's so we can incorporate patches (with https://github.com/openstack-packages/rdopkg)

**Comment by branto1** [comment]
lgtm, gitbuilders are happy

**Comment by ktdreyer** [comment]
Thanks guys!

**Pull request merged**